machine_learning_experiments/scripts/preprocess.py

```diff
@@ -13,10 +13,10 @@
     "frontend",
     "security",
     "devops",
-    "game-dev",
+    # "game-dev",
```

Comment:
I'm a little confused on our semantics around slugging here. I would expect `data-science` and `tech-writing` to be hyphenated if we're doing that, but it seems we're not consistent. Should we define how we're slugging things, so we can try to be consistent, as well as asking our users to be consistent.

Reply:
Sure. The important thing is that there has to be a keyword with a matching slug (we have a keyword "data science" and "datascience" so either way works).